src/Randomizer.App/Windows/OptionsWindow.xaml

```diff
@@ -184,6 +184,33 @@
                 </Grid>
               </controls:LabeledControl>
 
+              <controls:LabeledControl Text="Undo expiration time:"
+                                       ToolTip="Specifies how many minutes you have to undo a tracker action to limit accidentally undoing actions.">
+                <Grid>
+                  <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition Width="*" />
+                  </Grid.ColumnDefinitions>
+
+                  <TextBlock Grid.Column="0"
+                             Text="{Binding UndoExpirationTime, Mode=TwoWay, UpdateSourceTrigger=LostFocus}"
+                             Name="TextBlockUndoExpirationTime"
+                             Margin="0,3,7,3" />
+                  <Slider Grid.Column="1"
```

Comment:
I would just use a regular text input, having a maximum for this feels a little silly.

Reply:
Fair enough. I did it mostly to avoid bad values being inputted, but I added some super simple validation.